graphql-server/go-server/graph/schema/dataprocessing.graphqls

```diff
@@ -41,6 +43,8 @@ input AddDataProcessInput {
   data_process_config_info: [DataProcessConfigItem!]
   bucket_name: String!
   version_data_set_name: String!
+  namespace: String!
```

Comment:
@wangxinbiao  I don't see anywhere that. you use this `namespace` and `creator`.

Reply:
python code hasn't been submitted.